src/App.js

```diff
@@ -1,25 +1,26 @@
-import logo from './logo.svg';
 import './App.css';
+import { Component } from 'react';
+import Header from './Components/Header/Header';
+import MainContainer from './Components/MainContainer/MainContainer';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor() {
+    super();
+  }
+
+  postNewEmailSubscriber = (newEmailSubscriber) => {
+    //apiCalls POST function here with(newEmailSubscriber) passed in
+    //.then(data => if response is okay then show the user that their email is now subscribed)
```

Comment:
This looks good to me.  At this stage we have nothing that will need to be done when 'componentDidMount', but I'll admit, that tripped me up for a second! 😂

Reply:
I removed this function wireframe.